cli/cmd/snapshot.go

```diff
@@ -187,10 +187,14 @@ func (c *ServicedCli) cmdSnapshotList(ctx *cli.Context) {
 			for _, s := range snapshots {
 				//build a comma-delimited list of the tags
 				tags := strings.Join(s.Tags, ",")
+				snapshotID := s.SnapshotID
+				if s.Invalid {
+					snapshotID += " [INVALID]"
+				}
```

Comment:
Why are you setting this value again?  Wouldn't you have gotten this from the dao/model object?